sea-orm-cli/src/cli.rs

```diff
@@ -78,6 +78,12 @@ pub fn build_cli() -> App<'static, 'static> {
                         .help("The maximum amount of connections to use when connecting to the database.")
                         .takes_value(true)
                         .default_value("1")
+                ).arg(
+                    Arg::with_name("DATE_TIME_CRATE")
+                        .long("date-time-crate")
+                        .help("The datetime crate to use for generating entities.")
+                        .takes_value(true)
+                        .default_value("chrono")
```

Comment:
Hey @nahuakang, thanks for contributing! I guess we can constraint the input of `--date-time-crate`?
- https://docs.rs/clap/latest/clap/struct.Arg.html#method.possible_values

Reply:
Yes! Also just found out we're using `clap=^2.33.3` and the latest differs a bit in the `possible_values` usage (passing `&[&'b str]` vs. something implementing `IntoIter`).